creatures.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+$db = new PDO('mysql:host=db; dbname=AnimeCreatures', 'root', 'password');
+
+function display($db)
+{
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $query = $db->prepare('SELECT `name`, `movie`, `year`, `special_abilities`, `creepiness`, `img_dir` FROM `creatures`;');
+    $query->execute();
+    $collection = $query->fetchAll();
+    foreach ($collection as $creature) {
```

Comment:
Again, I'd recommend making this it's own function separate from the DB connection. That way each function is only in charge of doing one thing 😄